src/app/[locale]/rest-client/page.tsx

```diff
@@ -0,0 +1,140 @@
+'use client';
+
+import '@ant-design/v5-patch-for-react-19';
+import { useState } from 'react';
+import { useTranslations } from 'next-intl';
+import { Button, Card, Form, Input, Select, Space, Typography, Spin, Flex } from 'antd';
+import { SendOutlined } from '@ant-design/icons';
+import { Content } from 'antd/es/layout/layout';
+import { RequestBody, ApiResponse } from '@/types/interfaces';
+import { HttpMethod } from '@/types/types';
+import axios from 'axios';
+
+const { Title, Text } = Typography;
+const { TextArea } = Input;
+
+export default function RestClientPage() {
+  const t = useTranslations('RestClient');
+  const [form] = Form.useForm();
+  const [loading, setLoading] = useState(false);
+  const [response, setResponse] = useState<ApiResponse | null>(null);
+  const [error, setError] = useState<string | null>(null);
+
+  const handleSubmit = async (values: RequestBody) => {
+    setLoading(true);
+    setError(null);
+    setResponse(null);
+
+    try {
+      const response = await axios.post('/api/rest-client', values);
+
+      if (response.data.error) {
+        setError(response.data.error);
+      } else {
+        setResponse(response);
+      }
+    } catch (err) {
+      const errorMessage = err instanceof Error ? err.message : t('errorMessage');
+      setError(errorMessage);
+    } finally {
+      setLoading(false);
+    }
+  };
+
+  return (
+    <Content>
+      <Flex vertical align="center">
+        <Space direction="vertical" size="large" align="center">
+          <Title data-testid="rest-client-title">{t('title')}</Title>
+
+          <Card>
+            <Form
+              form={form}
+              layout="vertical"
+              onFinish={handleSubmit}
+              initialValues={{
+                method: HttpMethod.GET,
+                url: '',
+              }}
+            >
+              <Flex gap="large" align="end">
+                <Form.Item
+                  name="method"
+                  label={t('method')}
+                  rules={[{ required: true, message: t('methodRequired') }]}
+                >
+                  <Select data-testid="method-select">
+                    <Select.Option value={HttpMethod.GET}>{HttpMethod.GET}</Select.Option>
+                    <Select.Option value={HttpMethod.POST}>{HttpMethod.POST}</Select.Option>
+                    <Select.Option value={HttpMethod.PUT}>{HttpMethod.PUT}</Select.Option>
+                    <Select.Option value={HttpMethod.DELETE}>{HttpMethod.DELETE}</Select.Option>
+                    <Select.Option value={HttpMethod.PATCH}>{HttpMethod.PATCH}</Select.Option>
+                    <Select.Option value={HttpMethod.HEAD}>{HttpMethod.HEAD}</Select.Option>
+                    <Select.Option value={HttpMethod.OPTIONS}>{HttpMethod.OPTIONS}</Select.Option>
```

Comment:
1) Select.Option -> Option 
2) Use map for it. You need HttpMethod convert to array

This demo example
I did the conversion in the forehead, but it doesn't have to be that way. Or, for example, it is stored here. The key is to make a map.
Before rendering, I added this to get an array.
```const httpMethods = Object.values(HttpMethod);```

And use it
```suggestion
{ httpMethods.map(method => <Option value={method}>{method}</Option>) }
```

Reply:
Thank you, that is a great observation! I have changed it to a .map.